metrics/collectors/fibers.lua

```diff
@@ -0,0 +1,35 @@
+local fiber = require('fiber')
+local utils = require('metrics.collectors.utils');
+
+
+local function init()
+
+    local gauges = {}
+
+    global_metrics_registry:register_callback(function()
```

Comment:
In my opinion, try to avoid long closures. Define local function and register them - it's more readable for me.

Reply:
fixed